app/src/main/java/com/example/whereto/fragments/SeeOthersFragment.java

```diff
@@ -0,0 +1,105 @@
+package com.example.whereto.fragments;
+
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import androidx.annotation.Nullable;
+import androidx.fragment.app.Fragment;
+import androidx.lifecycle.Observer;
+import androidx.lifecycle.ViewModelProviders;
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
+
+import com.example.whereto.R;
+import com.example.whereto.models.Itinerary;
+import com.example.whereto.models.ItineraryAdapter;
+import com.example.whereto.models.SharedViewModel;
+import com.example.whereto.models.UserPreferences;
+import com.example.whereto.models.YelpBusiness;
+import com.parse.FindCallback;
+import com.parse.Parse;
+import com.parse.ParseException;
+import com.parse.ParseQuery;
+import com.parse.ParseRelation;
+import com.parse.ParseUser;
+
+import java.util.List;
+
+public class SeeOthersFragment extends Fragment {
+    private SharedViewModel model;
+    private UserPreferences userPreferences;
+    protected ItineraryAdapter adapter;
+    protected List<YelpBusiness> allItineraries;
+    RecyclerView rvItinerary;
+
+    public SeeOthersFragment() {
+    }
```

Comment:
Could you help me understand why this exists?

Reply:
I added it in because GSON requires an empty constructor!